bundles/org.openhab.binding.mystrom/src/main/java/org/openhab/binding/mystrom/internal/MyStromHandler.java

```diff
@@ -36,22 +35,20 @@
 import org.openhab.core.thing.Thing;
 import org.openhab.core.thing.ThingStatus;
 import org.openhab.core.thing.ThingStatusDetail;
-import org.openhab.core.thing.binding.BaseThingHandler;
 import org.openhab.core.types.Command;
 import org.openhab.core.types.RefreshType;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.gson.Gson;
-
 /**
  * The {@link MyStromHandler} is responsible for handling commands, which are
  * sent to one of the channels.
  *
  * @author Paul Frank - Initial contribution
+ * @author Frederic Chastagnol - Extends from new abstract class
  */
 @NonNullByDefault
-public class MyStromHandler extends BaseThingHandler {
+public class MyStromHandler extends MyStromAbstractHandler {
```

Comment:
Maybe rename to `MyStromPlugHandler` to keep the class naming consistent.

Reply:
Ok. I wasn't sure if I could rename the class as it was there before.

Reply:
That's not a problem because it's not exposed to the user and also an internal package.